Branch2018/test1/Program.cs

```diff
@@ -0,0 +1,338 @@
+        {
+            List<Customer> customersList = new List<Customer>();
+            List<Product> productsList = new List<Product>();
+            List<Seller> sellerList = new List<Seller>();
+            List<Sale> salesList = new List<Sale>();
+
+            int optionMenu = 0;
+            bool answerMenu;
+            int numberOpcion;
+            while (optionMenu != 6)
+            {
+                do
+                {
+                    Console.WriteLine("--------------------");
+                    Console.WriteLine("Billing Menu");
+                    Console.WriteLine("--------------------");
+                    Console.WriteLine("1. Sales Registration");
+                    Console.WriteLine("2. Customers Registration");
+                    Console.WriteLine("3. Seller Registration");
+                    Console.WriteLine("4. Products Registration");
+                    Console.WriteLine("5. Reports");
+                    Console.WriteLine("6. Exit");
+                    Console.WriteLine("Select an option");
+                    answerMenu= Int32.TryParse(Console.ReadLine(), out numberOpcion);
+                }   
+                    while (answerMenu == false);
+                optionMenu = numberOpcion;
+                switch (optionMenu)
+                {
+                    case 1: SalesRegistration(salesList); 
+                            break;
+
+                    case 2: CustomerRegistration(customersList); 
+                            break;
+
+                    case 3: SellerRegistration(sellerList); 
+                            break;
+
+                    case 4: ProductRegistration(productsList); 
+                            break;
+
+                    case 5:                
+                            {
+                                int opcion = 0;
+                                int answer;                            
+                                bool boolAnswer = true;
+                                while (opcion != 5)
+                                {
+                                    do
+                                    {
+                                        Console.WriteLine("--------------------");
+                                        Console.WriteLine("Reports Menu");
+                                        Console.WriteLine("1. Total Sales");
+                                        Console.WriteLine("2. Customers older to 55 years.");
+                                        Console.WriteLine("3. Customers who meet years in this month.");
+                                        Console.WriteLine("4. Products low stock.");
+                                        Console.WriteLine("5. Return.");
+                                        Console.WriteLine("Select an option");
+                                        boolAnswer = Int32.TryParse(Console.ReadLine(),out answer);
+                                    }   

```

Comment:
always after tryparse you have to put a condition to ask if the converssion was right. Don't forget TryParse give back a bool as return value.